integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2546,7 +2542,7 @@ msgstr "sichtbar"
 
 #: cms/models/languages/language_tree_node.py:29
 msgid "Defines whether the language is displayed to the users of the app"
-msgstr "Definiert, ob die Sprache den Nutzer:innen der App angezeigt wird"
+msgstr "Definiert, ob die Sprache den Konten der App angezeigt wird"
```

Comment:
```suggestion
msgstr "Definiert, ob die Sprache den Benutzer*innen der App angezeigt wird"
```
In this case "Konten" does not make sense, since the users of our app do not have logins etc...